pkg/client/k8s/k8s.go

```diff
@@ -202,6 +202,10 @@ func NewK8sClient(ns string) (*K8sClient, error) {
 	}, nil
 }
 
+func (k *K8sClient) GetOECS() *openebs.Clientset {
```

Comment:
The most cryptic method name for a first timer. Add a comment or expand the name.

Reply:
Sure